README.md

```diff
@@ -2,3 +2,29 @@
 
 Thanks to TenPlus1, blert2112, and taikedz
 
+
+## Grossam's fork changes log
+
+* 2019-07-31 :
+  * spawn.lua rewritten using mobs:spawn()
+  * spawn rules modified for more consistency (e.g. hedgehogs now spawn at night)
+  * some sounds and textures added
+  * dragon arrow bug corrected
+  * elemental an great drangons can fly in water (still take damages from it !)
+  * great dragons may spawn in caverealms
+* 2019-07-28 :
+  * some little bugs corrected
+  * dragon_normal described as 'minor dragons' and now they can spawn
+  * ice dragons are now named "Blue dragon"
+  * hatched great dragons are tamed now
+  * great dragons now need a diamond block nest (you've to deserve greatness !)
+* 2019-07-27 :
+  * Forked on https://git.fwhost.eu/Grossam/dmobs
+
+
+## Credits
+
+* dmobs_thunder.ogg : https://freesound.org/people/Josh74000MC/sounds/475094/
+* dmobs_wind.ogg : https://freesound.org/people/Pedaling%20Prince/sounds/338952/
+* dmobs_poison.ogg : https://freesound.org/people/qubodup/sounds/182789/
+* dmobs_wave.ogg : https://freesound.org/people/Kayyy/sounds/61011/
```

Comment:
You will need to add the explicit licenses to the `license.txt` for copyright compliance

`CC-0` for thunder, and `CC-BY 3.0` for the others.